GenderPayGap.WebUI/Startup.cs

```diff
@@ -125,9 +125,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                     o.Cookie.SecurePolicy = CookieSecurePolicy.Always; //Equivalent to <httpCookies requireSSL="true" /> from Web.Config
                     o.Cookie.HttpOnly = false; //Always use https cookies
                     o.Cookie.Domain = Global.ExternalHost.BeforeFirst(":"); //Domain cannot be an authority and contain a port number
-                    o.IdleTimeout =
-                        TimeSpan.FromMinutes(
-                            Program.MvcApplication.SessionTimeOutMinutes); //Equivalent to <sessionState timeout="20"> from old Web.config
```

Comment:
Does that mean that the previous value was 43200? How does that relate to the 20 in the comment?